convert2rhel/unit_tests/logger_test.py

```diff
@@ -33,52 +33,57 @@ def test_logger_handlers(tmpdir, caplog, read_std, is_py2, capsys):
     logger = logging.getLogger(__name__)
 
     # emitting some log entries
-    logger.info("Test info")
-    logger.debug("Test debug")
+    logger.info("Test info: %s", "data")
+    logger.debug("Test debug: %s", "other data")
 
     # Test if logs were emmited to the file
     with open(str(tmpdir.join(log_fname))) as log_f:
-        assert "Test info" in log_f.readline().rstrip()
-        assert "Test debug" in log_f.readline().rstrip()
+        assert "Test info: data" in log_f.readline().rstrip()
+        assert "Test debug: other data" in log_f.readline().rstrip()
 
     # Test if logs were emmited to the stdout
     stdouterr_out, stdouterr_err = read_std()
-    assert "Test info" in stdouterr_out
-    assert "Test debug" in stdouterr_out
+    assert "Test info: data" in stdouterr_out
+    assert "Test debug: other data" in stdouterr_out
 
 
 def test_tools_opts_debug(tmpdir, read_std, is_py2):
     log_fname = "convert2rhel.log"
     logger_module.setup_logger_handler(log_name=log_fname, log_dir=str(tmpdir))
     logger = logging.getLogger(__name__)
     tool_opts.debug = True
-    logger.debug("debug entry 1")
+    logger.debug("debug entry 1: %s", "data")
     stdouterr_out, stdouterr_err = read_std()
     # TODO should be in stdout, but this only works when running this test
     #   alone (see https://github.com/pytest-dev/pytest/issues/5502)
     try:
-        assert "debug entry 1" in stdouterr_out
+        assert "debug entry 1: data" in stdouterr_out
     except AssertionError:
         if not is_py2:
-            assert "debug entry 1" in stdouterr_err
+            assert "debug entry 1: data" in stdouterr_err
         else:
             # this workaround is not working for py2 - passing
             pass
     tool_opts.debug = False
-    logger.debug("debug entry 2")
+    logger.debug("debug entry 2: %s", "data")
```

Comment:
Note: Without the change to logger.py, this is the specific logger call that was broken (calling `logger.debug()` with a lazily evaluated string when `tool_opts.debug = False`)